.github/workflows/tests.yml

```diff
@@ -0,0 +1,20 @@
+name: tests 
+on: [push]
+jobs: 
+  test: 
+    name: Run unit tests 
+    runs-on: ubuntu-latest 
+    steps: 
+      - name: Checkout Master 
+        uses: actions/checkout@v2 
+      - name: Set up Python 3.7 
+        uses: actions/setup-python@v2 
+        with: 
+          python-version: '3.7.5' 
+      - name: Install dependencies 
+        run: | 
+          python -m pip install --upgrade pip 
+          pip install .
+          pip install pytest python-dotenv responses
```

Comment:
Why aren't we using `poetry` for stuff like this?

Reply:
The poetry install always times out on the actions :(

Reply:
That's such a bummer!

Reply:
Yes, and the biggest problem is that we'll probably need to find a way to make sure that the dev packages have the same version as in the lock file